### ISSUE TITLE
Filter: Announce number of filters applied when Filter button is focused

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#626]
## Description

When a filter is applied, we display the number filters applied beside the label. When using a screen reader, this value is not announced at all, leaving users unsure whether they have applied a filter or not. We should announce this information.

## Acceptance Criteria

- [ ] Announce the number of filters applied when Filter is in focus.
  - [ ] Explore using an `<aria-label=" Filter, 3 Filters applied">`

**Comment by bacitracin** [comment]
Issue is fixed on the live site, closing
